### DISCUSSION TITLE
Publish package to other service

### DISCUSSION THREAD

**Discussion by rhalaly** [semantic-release/semantic-release#1992] [Q&A]
Is this tool can publish the package to a service other than the NPM registry? For example, VSCode marketplace?

**Comment by travi** [comment]
does this information from our FAQ answer your question? https://semantic-release.gitbook.io/semantic-release/support/faq#can-i-use-semantic-release-to-publish-non-javascript-packages

**Reply by rhalaly** [reply]
Not really (maybe I missed something?). I have a **javascript** project **with** package.json. But I want to publish it to other services than NPM registry, like VSCode marketplace.

But when I configure the tool with `npx semantic-release-cli setup` it asks questions about the npm registry, which I don't have in that project.

**Reply by travi** [reply]
it is also possible to simply disable the publish step: https://semantic-release.gitbook.io/semantic-release/support/faq#can-i-skip-the-release-to-the-npm-registry

please also take a look at our list of plugins: https://semantic-release.gitbook.io/semantic-release/extending/plugins-list. it appears that https://github.com/felipecrs/semantic-release-vsce is targeted at publishing VSCode extensions, so it might be close to what you are looking for.
